backend/coreapp/compilers.py

```diff
@@ -893,6 +898,57 @@ def preset_from_name(name: str) -> Optional[Preset]:
     cc=MWCCARM_CC,
 )
 
+WATCOM_105_C = WatcomCompiler(
+    id="wcc10.5",
+    platform=MSDOS,
+    cc='${WINE} "${COMPILER_DIR}/binnt/wcc386.exe" -zq -i="..\\compilers\\wcc10.5\\h" -i="..\\compilers\\wcc10.5\\h\\nt" ${COMPILER_FLAGS} -fo"object.o" "code.c"',
```

Comment:
Can you replace the `"object.o"` with `"${OUTPUT}"` and `"code.c"` with `"${INPUT}"` to avoid the hardcoding?

Reply:
I tried with input/output variables but they are passed as a Unix path and watcom treats '/' as an optional way of indicating compiler options leading to errors of unknown options. I didn't see any obvious examples where this was worked around in the existing compilers but I'm open to suggestions on how to handle it better as I agree hard coding it isn't ideal.

Reply:
Ah fair enough, that's frustrating that wine doesn't automagically fix up your paths.